az-digital/az_quickstart
# Closes #1677: Update Text on Media Bottom Style

## Description
Update styles for light and dark text on media bottom styles

Tasks

- [x] Update bottom light styles
- [x] Update bottom dark styles
- [x] Add title alignment, update alignment options for title and content
- [x] ~~Change transparent class to pre-existing bg-transparent class~~
- [x] Add title heading level
   - [x] Add form field
   - [x] Change <h> tag based on selection
   - ~~[x] Apply `h2` class to <h> tag if `h3` `h4` `h5` or `h6` is selected~~
- [x] ~~Remove extraneous CSS~~

## Related issues
Closes #1677.
Follow-up issue regarding the transparent class: #1700.

## How to test


Add Text on Media paragraphs on a flexible page and confirm that all changes above have been implemented. See [this test page](https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-1683.probo.build/brian-test-page) as a starting point.

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- Patch release changes
- Minor release changes

### Drupal core
- Patch release changes
- Minor release changes

### Drupal contrib projects
- Patch release changes
- Minor release changes

## Checklist
